lib/l10n/app_hi.arb

```
@@ -0,0 +1,632 @@
+    "callDeclined": "कॉल अस्वीकार कर दिया गया",
+    "callDeclinedTo": "{username} ने आपकी कॉल को अस्वीकार कर दिया है।",
+    "@callDeclinedTo": {
+        "description": "Message when a call is declined",
+        "placeholders": {
+            "username": {
+                "type": "String",
+                "example": "john_doe"
+            }
+        }
+    },
+    "checkForUpdates": "अपडेट चेक करें",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)